agentMET4FOF_tutorials/tutorial_1_generator_agent.py

```diff
@@ -1,34 +1,4 @@
-from agentMET4FOF.agents import AgentMET4FOF, AgentNetwork, MonitorAgent
-from agentMET4FOF.streams import SineGenerator
-
-
-class SineGeneratorAgent(AgentMET4FOF):
-    """An agent streaming a sine signal
-
-    Takes samples from the :py:mod:`SineGenerator` and pushes them sample by sample
-    to connected agents via its output channel.
-    """
-
-    # The datatype of the stream will be SineGenerator.
-    _sine_stream: SineGenerator
-
-    def init_parameters(self):
-        """Initialize the input data
-
-        Initialize the input data stream as an instance of the
-        :py:mod:`SineGenerator` class
-        """
-        self._sine_stream = SineGenerator()
-
-    def agent_loop(self):
-        """Model the agent's behaviour
-
-        On state *Running* the agent will extract sample by sample the input data
-        streams content and push it via invoking :py:method:`AgentMET4FOF.send_output`.
-        """
-        if self.current_state == "Running":
-            sine_data = self._sine_stream.next_sample()  # dictionary
-            self.send_output(sine_data["quantities"])
+from agentMET4FOF.agents import AgentNetwork, MonitorAgent, SineGeneratorAgent
```

Comment:
We should still provide sufficient guidance on how to customize a generator agent. As discussed today between @bangxiangyong and @anupam-prasad we might provide information in tutorial 1 on how to set custom sine parameters (which we actually still need to introduce in the SineGeneratorAgent.